### PR TITLE
Add to vodka story

### PR DESCRIPTION
**Pull request by maxyatsenko**
Another action

### CODE REVIEW

**Review by maxyatsenko — COMMENTED** [review]
Please review this, @casey-collab.

**Comment by maxyatsenko** [comment]
Please review this, @casey-collab.

**Comment by casey-collab** [comment]
I was broken now I'm back. Can I review this? @casey-collab

File: russian/vodka/vodka.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Живительная влага мягко разливается по измученному организму и невероятным
 усилием воли получается наконец открыть глаза.
 Глаза открыты, в руках топор.
-[Я делаю бревно.](Brevno/brevno.md)
\ No newline at end of file
+[Я делаю бревно.](Brevno/brevno.md)
+[Я звоню маме.](../call-mom/call-mom.md)
```
Comment (casey-collab) [inline russian/vodka/vodka.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**